### PR TITLE
Add Google Login

### PR DESCRIPTION
**Pull request by RobertPlaiasu**
Be careful i made some changes to the database
If your tests include User model be sure everything works

### CODE REVIEW

File: Stoodle/app/User.php [hunk 1]
```diff
@@ -20,7 +20,7 @@ class User extends Authenticatable implements MustVerifyEmail
     protected $fillable = [
         'name', 'email', 'password','passion_intensity',
         'job' , 'social','profil_id' , 'passion_id',
-        'stress' , 'sport' , 'county_id' ,
+        'stress' , 'sport' , 'county_id' , 'google_id',
         'book_id','subject_id_1','subject_id_2','subject_id_3',
```
Comment (GrigoAleex) [inline Stoodle/app/User.php:24]: Why don t we just use `$guard = []`

File: Stoodle/composer.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
         "guzzlehttp/guzzle": "^6.3",
         "intervention/image": "^2.5",
         "laravel/framework": "^7.0",
+        "laravel/socialite": "^4.4",
```
Comment (GrigoAleex) [inline Stoodle/composer.json:17]: Will I need to run `compose install` or `composer update`?

File: Stoodle/app/User.php (hunk 1 above)
Comment (RobertPlaiasu) [reply Stoodle/app/User.php:24]: because it's not safe

File: Stoodle/composer.json (hunk 2 above)
Comment (RobertPlaiasu) [reply Stoodle/composer.json:17]: composer update

**Pull request merged**